routers/web/admin/runners.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2020 The Gitea Authors.
```

Comment:
2022

Reply:
And maybe "Copyright 2014 The Gogs Authors" should be removed? This is a new file that have nothing to do with Gogs, I think it was copied-pasted by accident.

Reply:
Yes